public/js/configs/feature.js

```diff
@@ -2,23 +2,19 @@
 
 let config = {};
 
-const developmentConfig = require("./development");
-
-// Hack, to let local development files be ignored by git
-let localDevelopmentConfig;
-try {
-  localDevelopmentConfig = require("./development.local");
-} catch (e) {
-  localDevelopmentConfig = {};
-}
-
-config = Object.assign({}, developmentConfig, localDevelopmentConfig);
+const developmentConfig = require("./development.json");
+config = Object.assign({}, developmentConfig);
+const originalConfig = Object.assign({}, developmentConfig);
```

Comment:
I dropped the local dev config file as we'll be going to a new option soon and it broke storybook